.github/workflows/run-tests.yml

```diff
@@ -17,39 +20,21 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python-version: [3.8, 3.9, '3.10', 3.11, 3.12]
+        python-version: [3.12]
```

Comment:
Actually, last comment, I know we say we support 3.12+ and Lambda doesn't currently support 3.13. Would it hurt to test 3.13 now so things are ready to go as soon as Lambda releases?

Reply:
Nevermind, they already released [3.13](https://aws.amazon.com/blogs/compute/python-3-13-runtime-now-available-in-aws-lambda/). Would that be worth adding?